src/smart-components/group/principal/principals.js

```diff
@@ -99,14 +99,25 @@ const GroupPrincipals = () => {
     </Fragment>
   );
 
+  const history = useHistory();
+
   const toolbarButtons = () => [
     ...(userIdentity && userIdentity.user && userIdentity.user.is_org_admin
       ? [
-          <Link to={`/groups/detail/${uuid}/members/add_members`} key="remove-from-group">
+          <Link to={`/groups/detail/${uuid}/members/add_members`} key="remove-from-group" className="pf-m-visible-on-md">
             <Button variant="primary" aria-label="Add member">
               Add member
             </Button>
           </Link>,
+          {
+            label: 'Add member',
+            props: {
+              className: 'pf-m-hidden-on-md',
+            },
+            onClick: () => {
+              history.push('/groups/detail/${uuid}/members/add_members');
```

Comment:
Shouldn't this be string literal instead of plain strin?
```suggestion
              history.push(`/groups/detail/${uuid}/members/add_members`);
```